src/tests/test_portal.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from duplocloud.errors import DuploError
+from duplocloud.client import DuploClient
+
+duplo, _ = DuploClient.from_env()
+
+class Testportal:
+
+  @pytest.mark.integration
+  def test_system_info(self):
+    r = duplo.load("system")
+    try:
+      lot = r("info")
```

Comment:
When I made `lot` it meant "list of tenants"
this can just be `info`

Reply:
Resolved